Project.toml

```diff
@@ -0,0 +1,21 @@
+name = "CutPruners"
+uuid = "7b25cd80-e7d3-5169-9be9-645fec4aeb50"
+repo = "https://github.com/cynddl/CutPruners.jl"
+version = "0.1.3"
+
+[deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
```

Comment:
Remove Compat, no need for it anymore as we drop Julia v0.6 and v0.7

Reply:
I would add `LinearAlgebra` and `SparseArrays`  as well